modules/images-imagestream-configure.adoc

```diff
@@ -37,7 +37,7 @@ status:
       dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d <3>
       generation: 2
       image: sha256:909de62d1f609a717ec433cc25ca5cf00941545c83a01fb31527771e1fab3fc5 <4>
-    - created: 2017-09-29T13:40:11Z
+    - created: 2017-09-01T13:40:11Z
```

Comment:
Above the doc `creationTimestamp: 2017-09-29T13:33:49Z` need to update too.

Reply:
Sorry, I'd lost track of this PR.  4a40c43f5a -> 5780bf90d7 rebases onto master and bumps `creationTimestamp` to `2017-09-01T13:40:00Z` to be slightly before the first `created` timestamp.